### DISCUSSION TITLE
forcing router

### DISCUSSION THREAD

**Discussion by michal-bartak** [Koenkk/zigbee2mqtt#11919] [Devices]
Is there any way to somehow force device to use particular router?

I have lidl door sensor working fine for several months, through Hue bulb as router. And it worked instantly. 
Recently all of sudden it switched to coordinator resulting in very low LQI and very long reaction times (from 30 secs to never). 

Is there any way how to tell the device to rescan/reconnect, or prefer other router if available?
Re-pairing is cumbersome. Basically because it will activate tamper which in turn requires zigbee2mqtt to remove this device and add again (BTW the same for battery :( )

**Comment by Koenkk** [comment]
You cannot force a device to use a certain router, what you can do is pair the device via a certain router and hope it stick to that. To pair a device via a specific router repair it while joining is only enabled via a specific router (in the z2m frontend click the drop down arrow next to permit join)

**Reply by michal-bartak** [reply]
thank you this info. Didn't know about such a feature (although the hope is what is still needed  ;) )
